src/frame/js/actions/conversation.js

```diff
@@ -577,10 +578,11 @@ export function startConversation() {
                 });
         } else {
             promise = dispatch(http('POST', `/apps/${appId}/appusers`, {
-                ...pendingAttributes,
+                ...pendingUserProps,
                 userId,
                 client: getClientInfo(appId)
             }));
+            dispatch(resetPendingUserProps());
```

Comment:
Should `resetPendingUserProps` wait for the POST to finish?

Reply:
hmm, right, otherwise the `getUser()` behavior would be a bit weird during that time.